devtool

```diff
@@ -33,7 +33,7 @@ lint() {
     echo "===================="
     echo "Note: This only syncs poetry.lock with pyproject.toml.  If you wish to update all downstream dependencies, run 'poetry update'"
 
-    install_poetry_version 1.2.0
+    install_poetry_version 1.8.2
```

Comment:
Non-blocking (can be addressed in a future PR since I already approved this one): we should call `install_poetry` here instead so we don't have to keep track of the poetry version in two places.